### ISSUE TITLE
OCR Calibration issue with 3 monitors

### ISSUE THREAD

**Issue by noots** [stringandstickytape/RegulatedNoise#46]
Using an Nvidia Titan video card with NV surround enabled to create one virtual resolution of 7680 x 1600.  Screenshots are clear but I cannot calibrate because some pointers are locked to certain coords.  Example: pointer 2 stays at 2469;x.  X moves, but 2469 stays as a constant.  I cannot select the bottom right of the station name.  Is this intentional?

Watched folder does not end up performing any OCR keeping my DB completely empty.

**Comment by stringandstickytape** [comment]
Yes, some of the points "snap" to match other points. Looks like your
screen config creates a wide enough station name to need more space.
Please post a sample Screenshot_????.BMP and I'll try to take a look at the
weekend when I might have some time :-/

If you're feeling ambitious, as a workaround you can directly edit your
calibration.txt file and as long as you don't then recalibrate, the change
should stick. If that works let me know, it will help me fix it.

**Comment by noots** [comment]
![2015-01-19 16_47_52-elite - dangerous client](https://cloud.githubusercontent.com/assets/10602647/5810422/0fa9a386-9ffb-11e4-8688-3b89f081a64e.png)

**Comment by noots** [comment]
I will load the file on photoshop to get x, y coords and plug them in manually in the calibration.txt file.  I'll post an update if that works or not.  

Is there a way to zoom in and out of the main graphic (left side of the UI)?

**Comment by noots** [comment]
Here is a copy of the full image capture.

https://www.dropbox.com/s/n6p00m6z1obes0v/Screenshot_0031.bmp?dl=0

**Comment by noots** [comment]
Manual modification of the calibration.txt file worked like a charm!

2630;150;2970;190;2620;400;3128;400;3247;400;3370;400;3495;400;3652;400;3745;400;3902;400;4020;400;2628;1394;
